### ISSUE TITLE
Device will be busy when creating file system on a new partition without reboot OS

### ISSUE THREAD

**Issue by TsutomuNakamura** [tsuna-server/build-server-ansible#104]
```
root@dev-storage01:~# gdisk /dev/vdb
GPT fdisk (gdisk) version 1.0.8

...

Command (? for help): n
Partition number (1-128, default 1):
First sector (34-16777182, default = 2048) or {+-}size{KMGTP}:
Last sector (2048-16777182, default = 16777182) or {+-}size{KMGTP}:
Current type is 8300 (Linux filesystem)
Hex code or GUID (L to show codes, Enter = 8300):
Changed type of partition to 'Linux filesystem'

Command (? for help):

Command (? for help): w

Final checks complete. About to write GPT data. THIS WILL OVERWRITE EXISTING
PARTITIONS!!

Do you want to proceed? (Y/N): Y
OK; writing new GUID partition table (GPT) to /dev/vdb.



The operation has completed successfully.

root@dev-storage01:~# mkfs.xfs /dev/vdb1
mkfs.xfs: cannot open /dev/vdb1: Device or resource busy
```

**Comment by TsutomuNakamura** [comment]
Commands below might cause this issue. If I want to create new partitions and format them without reboot OS, we should consider the sequence.
```
pvcreate (device)
vgcreate cinder-volumes (device)
```

**Comment by TsutomuNakamura** [comment]
This issue relates another issue #102.

**Comment by TsutomuNakamura** [comment]
This issue has solved `00175df` already.
